packages/adapters/watcher/src/types.ts

```diff
@@ -81,6 +90,9 @@ export const WatcherConfigSchema = Type.Object({
   twilioToPhoneNumbers: Type.Optional(Type.Array(Type.String())),
   telegramApiKey: Type.Optional(Type.String()),
   telegramChatId: Type.Optional(Type.String()),
+  keybaseUser: Type.Optional(Type.String()),
+  keybaseKey: Type.Optional(Type.String()),
+  keybaseChannel: Type.Optional(KeybaseChannelSchema),
 });
```

Comment:
This config should be set up on the watcher agent as well. 

https://github.com/connext/nxtp/blob/415dfad60534d11af498b74e87a64135a2235f1d/packages/agents/watcher/src/config.ts#L11